src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py

```diff
@@ -238,6 +238,26 @@ def load_arguments(self, _):
     with self.argument_context('aks remove-dev-spaces') as c:
         c.argument('prompt', options_list=['--yes', '-y'], action='store_true', help='Do not prompt for confirmation')
 
+    # OpenShift command argument configuration
+    with self.argument_context('openshift') as c:
+        c.argument('resource_name', name_type, help='Name of the managed OpenShift cluster.',
+                   completer=get_resource_name_completion_list('Microsoft.ContainerService/OpenShiftManagedClusters'))
+        c.argument('name', name_type, help='Name of the managed OpenShift cluster.',
+                   completer=get_resource_name_completion_list('Microsoft.ContainerService/OpenShiftManagedClusters'))
+        c.argument('compute_count', options_list=['--compute-count', '-c'], type=int, default=4)
+        c.argument('tags', tags_type)
+
+    with self.argument_context('openshift create') as c:
+        c.argument('name', validator=validate_linux_host_name)
+        c.argument('compute_vm_size', options_list=['--compute-vm-size', '-s'])
+        c.argument('fqdn', required=True)
+        c.argument('aad_client_app_id')
+        c.argument('aad_client_app_secret')
+        c.argument('aad_tenant_id')
+        c.argument('vnet_cidr')
+        c.argument('subnet_cidr')
+        c.argument('vnet_peer_id')
```

Comment:
L254-259: None of these are necessary because they don't do anything. You can safely remove them.